pkg/packagekit/autoupdates.jsx

```diff
@@ -337,6 +337,144 @@ class Dnf5Impl extends ImplBase {
     }
 }
 
+class AptImpl extends ImplBase {
+    async getConfig() {
+        this.packageName = "unattended-upgrades";
+
+        try {
+            await cockpit.spawn(["dpkg", "-l", this.packageName], { err: "ignore" });
+            this.installed = true;
+        } catch (ex) {
+            this.installed = false;
+            debug("apt getConfig: not installed:", ex);
+            return;
+        }
+
+        try {
+            // Check timer status and configuration
+            const timerOutput = await cockpit.script(
+                "set -eu; " +
+                "if systemctl --quiet is-enabled apt-daily-upgrade.timer 2>/dev/null; then echo enabled; fi; " +
+                "systemctl cat apt-daily-upgrade.timer 2>/dev/null | grep '^OnCalendar=' | tail -n1 || true; ",
+                [], { err: "message" });
+
+            this.enabled = (timerOutput.indexOf("enabled\n") >= 0);
+
+            // Parse timer schedule
+            const calIdx = timerOutput.indexOf("OnCalendar=");
+            if (calIdx >= 0) {
+                const calendarSpec = timerOutput.substring(calIdx).split('\n')[0].split("=")[1];
+                this.parseCalendar(calendarSpec);
+            } else {
+                if (timerOutput.indexOf("InactiveSec=1d\n") >= 0)
+                    this.day = this.time = "";
+                else if (this.installed)
```

Comment:
This should be unreachable, we return directly if it is not installed. (same issue is in the dnf5 implementation)